### ISSUE TITLE
Webpack Support

### ISSUE THREAD

**Issue by alexheyd** [alexheyd/atom-jump-to-import#26]
From @alexisab:

And will it be possible to consider aliases defined, for example, in `webpack.config.js` ? 

I've defined some aliases in my webpack config : 
```
resolve: {
        alias: {
            '~': path.resolve(__dirname, env.NODE_ENV === 'production' ? './server-dist' : './src'),
            'style': path.resolve(__dirname, (env.NODE_ENV === 'production' ? './server-dist' : './src') + '/style'),
        }
    },
```

so in my code I've many `import`s  like `import { ProjectMap } from '~/components/project-map'`.

**Comment by alexheyd** [comment]
@alexisab Will be adding this at a later date, `webpack.config.js` has some special rules around aliases in the form of `xyz$`.

It also defaults to looking for modules in `node_modules` and the examples they have listed [here](https://webpack.github.io/docs/configuration.html#resolve) don't all make sense to me so I'll need to set up a `webpack` project and confirm the behavior.

**Comment by alexheyd** [comment]
Support for Webpack's `resolve.alias` has been added. Released in `v3.6.0`
Please note that this ONLY supports `resolve.alias` and none of Webpack's other Module API.

Also, any changes to `webpack.config.js` requires Atom to be reloaded before the changes are picked up (working on a fix).
